cvrf2csaf/section_handlers/vulnerability.py

```diff
@@ -91,6 +105,85 @@ def _handle_remediations(root_element):
 
         return remediations
 
+    @staticmethod
+    def _base_score_to_severity(base_score):
+        if base_score >= 9.0:
+            return 'CRITICAL'
+        elif base_score >= 7.0:
+            return 'HIGH'
+        elif base_score >= 4.0:
+            return 'MEDIUM'
+        elif base_score >= 0.1:
+            return 'LOW'
+        else:
+            return 'NONE'
+
+    def _parse_score_set(self, score_set_element, mapping, version, json_property, product_status):
+        """Parses ScoreSetV2 or ScoreSetV3 element."""
+
+        # Parse all input elements except ProductID
+        # note: baseScore is always present since it's mandatory for the input
+        cvss_score = {csaf: getattr(score_set_element, cvrf).text for cvrf, csaf in mapping.items() if hasattr(score_set_element, cvrf)}
+        
+        cvss_score['baseScore'] = float(cvss_score['baseScore'])
+        cvss_score['baseSeverity'] = self._base_score_to_severity(cvss_score['baseScore'])
```

Comment:
Good catch. But this should only be done, if the major version is 3 as [CVSS v2](https://www.first.org/cvss/cvss-v2.0.json) didn't had the `baseSeverity` element.

Reply:
Ok, will fix + use bisect.